platform/ios/PolyPodApp/PodApi/domConsole.js

```diff
@@ -1,11 +1,15 @@
+function postLogMessage(message) {
+    window.parent.postMessage(
+        { command: "log", text: JSON.stringify(message) },
+        "*"
+    );
+}
+
 const console = {
     log(msg) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(msg) }, "*");
+        postLogMessage(msg);
     },
     error(msg) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(msg) }, "*");
+        postLogMessage(msg);
     },
-    dir(object) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(object) }, "*");
-    }
```

Comment:
Not a good idea to eliminate `console.dir` IMHO, it may be used. And we should aim towards supporting as much of the Console API as feasible IMHO.

Reply:
I'm not sure it does; I checked it through and it's not used anywhere. All the code that used it was removed at one point or another. And since this is posting a message which is not directed, it's bound to keep raising issues with snyk and others.